jasmine/spec/invertedIndexSpec.js

```diff
@@ -1,62 +1,53 @@
+'use strict';
+
 var index = new Index();
+index.indexObject = null;
 
 beforeEach(function(done) {
-  index.createIndex('books.json').then(function () { 
+  index.createIndex('books.json').then(function () {
     done();
-  })
-  
+  });
+
 });
 
-describe("Index", function() {
-    it("should read the JSON file and assert it's not empty", function() {
-        // index.createIndex('../jasmine/books.json');
-        console.log("hurrray", index)
-        console.log("docs", index.books)
-        expect(index.books.length).toBeGreaterThan(0);
+describe('Index', function() {
+  it('should read the JSON file and assert it is not empty', function() {
+    expect(index.books.length).toBeGreaterThan(0);
 
     });
 
 });
 
 
-
-describe("populate Index", function() {
-    // it('Does something', function() {
-    //     console.log('SOMETHING HAPPENING');
-    //     expect(2).toBe(2);
-    // });
-    it("should check that index is created once the JSON file has been read", function() {
-        console.log(index.indexObject)
-        expect(index.indexObject.length).not.toBe(0);
-
+describe('populate Index', function() {
+   
+  it('should check that index is created once the JSON file has been read', function() {
+    expect(index.indexObject.length).not.toBe(0);
     });
 
-    it("should check index maps the string keys to the correct objects in the JSON array.", function() {
-        index.getIndex()
-        expect(index.indexObject["powerful"]).toEqual([ 1 ]);
-        expect(index.indexObject["falls"]).toEqual([ 0 ]);    
+  it('should check index maps the string keys to the correct objects in the JSON array.', function() {
+    index.getIndex();
+    expect(index.indexObject.'powerful').toEqual([ 1 ]);
```

Comment:
**Code Quality - Critical**

Use the array notation to access the index like so: `index.indexObject['powerful']`